src/model/cloud-runner/services/cloud-runner-custom-steps.ts

```diff
@@ -77,13 +143,13 @@ export class CloudRunnerCustomSteps {
     aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID --profile default
```

Comment:
When I see commands in a github action step, I'm always wondering if it's better to have them in the yaml or expose a shell script so we gain syntax highlighting and we can run shellcheck or other tools like that against them. _just thinking out loud here_

Reply:
Will resolve through referenced issue. Might address post-merge.